src/tools/splash2txt/splash2txt.cpp

```diff
@@ -159,8 +159,9 @@ bool parseOptions( int argc, char** argv, ProgramOptions &options )
         }
 
     }
-    catch ( boost::program_options::error e )
+    catch ( const boost::program_options::error& )
     {
+        errorStream << desc << "\n";
```

Comment:
why did you add a error stream output here?
(not mentioned in the pull request description)